assets/sass/base/_page.scss

```diff
@@ -1,7 +1,7 @@
 ///
-/// Hyperspace by HTML5 UP
-/// html5up.net | @ajlkn
-/// Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
+/// Hyperspace by TheMaestro0
+/// customtechenterprize.crypto
+/// Free for personal use under the +NIGGER License (https://plusnigger.autism.exposed)

```

Comment:
I think using the +NIGGER License is fine. Especially when corporate theft by progressive devs is rampant.